database/README.md

```diff
@@ -301,3 +302,30 @@ new FirebaseIndexListAdapter<Chat>(this, Chat.class, android.R.layout.two_line_l
 ```
 
 `keyRef` is the location of your keys, and `dataRef` is the location of your data.
+
+### A note on ordering
+
+The order in which your receive your data depends on the order from `keyRef`, not `dataRef`:
+```json
+{
+  "data": {
+    // This order doesn't matter, the order is taken from keys/(user1 or user2).
```

Comment:
I'm not sure what to do about these, I want to add comments but json doesn't like that.